app/controllers/event/approval_controller.rb

```diff
@@ -0,0 +1,36 @@
+# encoding: utf-8
+
+#  Copyright (c) 2017, Pfadibewegung Schweiz. This file is part of
+#  hitobito_pbs and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_pbs.
+
+class Event::ApprovalController < CrudController
+
+  def self.model_class
+    Event::Approval
+  end
+
+  private
+
+  def entry
```

Comment:
Statt ``entry``, solltest du bei einem ``CrudController`` ``build_entry`` überschreiben. Aber die Frage ist eher, ob in diesem Fall ein ``CrudController`` angebracht ist, da es nicht sehr "CRUDy" ist (persistiert wird es über den Approver und alle anderen Actions gibt es nicht).